### DISCUSSION TITLE
Internal connection to paperless

### DISCUSSION THREAD

**Discussion by Sblop** [SecureAI-Tools/SecureAI-Tools#92] [Ideas]
Hi, Thanks for a great project.
I face problems in connecting directly to paperless webserver, using only the internal docker container name. I have a feeling that secureAI-Tools only can connect using a eg paperless.domain.tdl, but I wish to connect to paperless-webserver:port.
As it is gernerally not recommended to share paperless directly to the internet, without using something like authelia, I am unable to connect to paperless.

**Comment by JayNakrani** [comment]
Here is one way to achieve this using docker's host-ip: 
1. Run paperless-ngx and SecureAI-Tools both on one host machine and expose paperless-ngx port to the host (w/o opening it up to the rest of the internet).
2. Then point SecureAI-Tools to paperless-ngx's [docker's special host-ip](https://stackoverflow.com/a/24326540) and port. i.e. instead of using `paperless.domain.tdl` in the UI, use `host.docker.internal:<port>` (`host.docker.internal` is the host's special IP in docker-networking)

**Comment by Sblop** [comment]
Hi Jay, 
Thanks. This solved the problem. I never tried this approach, only the direct hostname of the container. Thanks.
